service/src/main/java/uk/gov/hmcts/reform/fpl/model/Orders.java

```diff
@@ -16,6 +18,7 @@
     @NotNull(message = "Select at least one type of order")
     @Size(min = 1, message = "Select at least one type of order")
     private final List<OrderType> orderType;
+    private final List<EmergencyProtectionOrdersType> emergencyProtectionOrders;
     private final String directions;
-    private final List<String> emergencyProtectionOrderDirections;
```

Comment:
@akomisarek does this break things? here's the ccd
```
  {
    "LiveFrom": "01/01/2017",
    "ID": "Orders",
    "ListElementCode": "emergencyProtectionOrderDirections",
    "FieldType": "MultiSelectList",
    "FieldTypeParameter": "EmergencyProtectionOrderDirectionType",
    "ElementLabel": "Do you need any of these directions?",
    "FieldShowCondition": "orderType CONTAINS \"EMERGENCY_PROTECTION_ORDER\"",
    "SecurityClassification": "Public"
  },
```

Reply:
if your enum maps to the same strings, no it won't break